x-pack/plugins/enterprise_search/public/plugin.ts

```diff
@@ -213,6 +214,27 @@ export class EnterpriseSearchPlugin implements Plugin {
       },
     });
 
+    core.application.register({
+      id: SEARCH_EXPERIENCES_PLUGIN.ID,
+      title: SEARCH_EXPERIENCES_PLUGIN.NAME,
+      euiIconType: ENTERPRISE_SEARCH_OVERVIEW_PLUGIN.LOGO,
+      appRoute: SEARCH_EXPERIENCES_PLUGIN.URL,
+      category: DEFAULT_APP_CATEGORIES.enterpriseSearch,
+      mount: async (params: AppMountParameters) => {
+        const kibanaDeps = await this.getKibanaDeps(core, params, cloud);
+        const { chrome, http } = kibanaDeps.core;
+        chrome.docTitle.change(SEARCH_EXPERIENCES_PLUGIN.NAME);
+
+        await this.getInitialData(http);
```

Comment:
Does this plugin need this data?

Reply:
Great question, most likely no. This is a bad copy + paste on my part.

Reply:
I was just tracing the code. I think it's OK to leave it here. It looks like it's part of the standard plugin boilerplate for all of our plugins. The data fetched from there actually gets passed through to our shared [renderApp](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/index.tsx#L42) which I think requires that data.